pkg/prowgen/prowgen.go

```diff
@@ -193,14 +193,18 @@ func generatePresubmitForTest(jobBaseBuilder *prowJobBaseBuilder, name string, i
 
 	shortName := info.TestName(name)
 	base := jobBaseBuilder.Rehearsable(!opts.disableRehearsal).Build(jc.PresubmitPrefix)
+	rerunCommand := prowconfig.DefaultRerunCommandFor(shortName)
```

Comment:
I'm on the fence about this change, but you could also put the logic inside `prowconfig.DefaultRerunCommandFor`. This is the only usage.

Reply:
Unfortunately `prowconfig.DefaultRerunCommandFor` is upstream

Reply:
ah, it is. makes sense then.